redis/client.py

```diff
@@ -785,8 +786,20 @@ def client_kill(self, address):
         "Disconnects the client at ``address`` (ip:port)"
         return self.execute_command('CLIENT KILL', address)
 
-    def client_list(self):
+    def client_list(self, _type=None):
+        """
+        Returns a list of currently connected clients.
+        If type of client specified, only that type will be returned.
+        :param _type: optional. one of the client types (normal, master,
+         replica, pubsub)
+        """
         "Returns a list of currently connected clients"
+        if _type is not None:
+            if _type not in ['normal', 'master', 'replica', 'pubsub']:
```

Comment:
Perhaps `if str(_type).lower() not in ['normal', 'master', 'replica', 'pubsub']:`?

Reply:
Agreed with Itamar.

Reply:
Resolved via 0adcc53